baseapp/testproject/settings.py

```
@@ -0,0 +1,55 @@
+from baseapp_core.tests.settings import *  # noqa
+from django.utils.translation import gettext_lazy as _
+
+# Application definition
+INSTALLED_APPS += [
```

Ruff (0.7.0)

`INSTALLED_APPS` may be undefined, or defined from star imports

(F405)